docs/docs/jupyter-integration.ipynb

```diff
@@ -0,0 +1,306 @@
+   "execution_count": null,
+   "id": "c1bbee21-3241-41d7-a7e2-d7551a47a5bc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dp.Group(\n",
+    "    dp.BigNumber(\n",
+    "        heading=\"Percentage points\",\n",
+    "        value=\"84%\",\n",
+    "        change=\"2%\",\n",
+    "        is_upward_change=True,\n",
+    "    ),\n",
+    "    dp.BigNumber(\n",
+    "        heading=\"Points\",\n",
+    "        value=\"1234\",\n",
+    "        change=\"200\",\n",
+    "        is_upward_change=False,\n",
+    "    ),\n",
+    "    columns=2,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aac7e2d0-5f5d-41d5-ae6d-dee5628cc872",
+   "metadata": {},
+   "source": [
+    "## Notebook to Data App? One line of code."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "249ca3b0-1c88-486b-8fdb-5e87b37d553d",
+   "metadata": {},
+   "source": [
+    "We've built our data app, but now we want to share it! \n",
+    "\n",
+    "Some recipients will be comfortable with an `.ipynb` file, and in those instances, you could just send your notebook over and still benefit from Datapane's enhancements.\n",
+    "\n",
+    "But in most cases, you want to turn your notebook into something presentable and accessible. With Datapane, all it takes is a single line of code.\n",
+    "\n",
+    "```python\n",
+    "app = dp.App(blocks=dp.cells_to_blocks(Out))\n",
+    "```\n",
+    "\n",
+    "!!! warning\n",
+    "    Don't forget to save your notebook before calling `cells_to_blocks`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8c0c1e4e-9fa8-4161-8979-daacd0096a63",
+   "metadata": {},
+   "source": [
+    "From here, you can save your shareable app locally.\n",
+    "\n",
+    "```python\n",
+    "app.save(\"my_app.html\")\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a320e533-32b1-4bbb-9014-821d7942fbe4",
+   "metadata": {},
+   "source": [
+    "Or upload it to Datapane Cloud and get a [shareable link](https://cloud.datapane.com/reports/0kzdJR3/my-app/).\n",
+    "\n",
+    "```python\n",
+    "app.upload(\"My App\")\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3e52ad2e-caba-4891-bf3f-667b97d61160",
+   "metadata": {},
+   "source": [
+    "### Opt-in or opt-out"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0e639397-eb30-495c-85e5-b767ebeee0b7",
+   "metadata": {},
+   "source": [
+    "By default, notebook-to-app conversion will include all markdown and supported cell output. In this mode, you can explicitly exclude cells with the cell tag `dp-exclude`. \n",
+    "\n",
+    "In some cases, it may be preferable to only include cells of interest. This can be achieved by setting `opt_out` to `False`, e.g.:\n",
+    "\n",
+    "```python\n",
+    "app = dp.App(blocks=dp.cells_to_blocks(Out, opt_out=False))\n",
+    "```\n",
+    "\n",
+    "In this mode, you can explicitly include supported cells with the cell tag `dp-include"
```

Comment:
Missing end of code tag, and can we have more example here? I think this is a key feature